### PR TITLE
Various fixes

### LINKED ISSUE #1890: Method name error of Flux object in python client docs.
_Method name error of Flux object._

##### Relevant URLs
- _https://docs.influxdata.com/influxdb/v2.0/tools/client-libraries/python/#query-data-from-influxdb-with-python_

##### What products and version are you using?
InfluxDB 2.0.1 (git: 37cc047133) build_date: 2020-11-11T03:53:31Z

##### Detail

The docs state that `get_values()` returns the actual field value, but it reports error: `AttributeError: 'FluxRecord' object has no attribute 'get_values'`. Actually it should be `get_value()`

### LINKED ISSUE #1891: InfluxDB v2 plugin documentation - Wrong link
It seems that "InfluxDB V2 Listener"  plugin summary and link to its gihub repository is wrong.

Point to https://github.com/influxdata/telegraf/blob/release-1.16/plugins/inputs/influxdb_listener/README.md
instead of https://github.com/influxdata/telegraf/blob/release-1.16/plugins/inputs/influxdb_v2_listener/README.md

##### Relevant URLs
https://docs.influxdata.com/telegraf/v1.16/plugins/

### PR DESCRIPTION
**Pull request by sanderson**
Related to #1892 
Closes #1891 
Closes #1890 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: data/telegraf_plugins.yml [hunk 1]
```diff
@@ -638,21 +638,22 @@ input:
...
-      </div>
     introduced: 1.8.0
     tags: [linux, macos, windows, data-stores]
 
   - name: InfluxDB Listener
     id: influxdb_listener
     description: |
       The InfluxDB Listener input plugin listens for requests sent
-      according to the [InfluxDB HTTP API](https://docs.influxdata.com/influxdb/latest/guides/writing_data/).
+      according to the [InfluxDB HTTP API](/influxdb/v1.8/guides/writing_data/).
```
Comment (kelseiv) [inline data/telegraf_plugins.yml:656]: /write_data for good measure

File: data/telegraf_plugins.yml [hunk 2]
```diff
@@ -673,26 +674,16 @@ input:
...
-      > This plugin was previously known as `http_listener`.
-      > To send general metrics via HTTP, use the [HTTP Listener v2 input plugin](#http_listener_v2) instead.
-      >
-      > This plugin is compatible with **InfluxDB 2.x** only.
+      for the HTTP `/api/v2/write` endpoint of the InfluxDB HTTP API.
 
-      The `/write` endpoint supports the `precision` query parameter and can be set
-      to one of `ns`, `u`, `ms`, `s`, `m`, `h`.  All other parameters are ignored and
+      The `/api/v2/write` endpoint supports the `precision` query parameter and can be set
+      to one of `ns`, `u`, `ms`, or `s`.  All other parameters are ignored and
```
Comment (kelseiv) [inline data/telegraf_plugins.yml:685]: to `ns`...

...or `s`. Other

File: content/influxdb/v2.0/reference/cli/influx/write/_index.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: influx write
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/cli/influx/write/_index.md:2]: As a best practice globally, we may want to start including overview links at top, so if folks land on this page, they immediately know examples exist.

File: content/influxdb/v2.0/reference/cli/influx/write/_index.md [hunk 4]
```diff
@@ -54,3 +65,138 @@ influx write [command]
...
+
+##### Write annotated CSV data from multiple sources
+```sh
+influx write \
+  --bucket example-bucket \
+  --file path/to/data-1.csv \
+  --url https://example.com/data-2.csv
+```
+
+##### Prepend CSV data with annotation headers
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/cli/influx/write/_index.md:193]: Prepend CSV data with annotation headers (and write data from a file?...or can folks do multiple options here?)

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/cli/influx/write/_index.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/cli/influx/write/_index.md:2]: Agree. There's actually a way to have Hugo do this. I'm going to leave this as is for now and incorporate that work in another PR.

File: content/influxdb/v2.0/reference/cli/influx/write/_index.md (hunk 4 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/cli/influx/write/_index.md:193]: They can do multiple. This specific example just shows how to do it with a file.

**Pull request merged**